docs/source/tutorial-model-registry.rst

```diff
@@ -0,0 +1,110 @@
+
+Register a model with the Model Registry
+----------------------------------------
+
+You can register a model with the Model Registry using the MLflow UI, the MLflow API, or the MLflow CLI. If you've completed the :ref:`quickstart` or :ref:`tutorial-tracking-ui`, you already have some experience with the MLflow UI and have even registered a model in the Model Registry.
+
+Get your experiment's ``Run ID``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The MLflow Model Registry works with the MLflow Tracking component. As a result, the Model Registry uses the same experiment ``Run ID`` to identify a model. Using the skills you learned in :ref:`tutorial-tracking-ui`, choose an experiment that you want to register and deploy. 
+
+In the MLflow UI, the Run ID is prominently located in the upper left position of the run's details page:
+
+.. figure:: _static/images/tutorial-model-registry/run-details.png
+   :alt: screenshot showing the location of Run ID in the MLflow UI
+
+You may also retrieve the Run ID using the MLflow API. For example, the following code snippet retrieves the Run ID for the run with the lowest ``val_rmse`` error:
+
+.. code-block:: python
+
+    import mlflow
+    import os
+
+    tracking_uri = os.environ.get("MLFLOW_TRACKING_URI") or "http://localhost:5000"
+    mlflow.set_tracking_uri(tracking_uri)
+
+    experiment_ids = [0] # 'Default' experiment
+    order_pred = ["metrics.val_rmse asc"] # sort by lowest validation error
+    runs_df = mlflow.search_runs(experiment_ids=experiment_ids, order_by=order_pred, max_results=10)
+    run_id = runs_df.loc[0]["run_id"]
+    print(f"Run run_id: {run_id} had lowest validation error: {runs_df.loc[0]['metrics.val_rmse']}")
+
+Register the run's model in the Model Registry
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Once you have the Run ID, you can register the model in the Model Registry. 
+
+In the MLflow UI, you can register a model by clicking the ``Register Model`` button in the upper right corner of the run's page. This is marked with the red **2** in the previous screenshot.
+
+You may also register a model with the MLflow API's ``mlflow.register_model`` function. You must set the ``model_uri`` and ``name`` parameters. The ``model_uri`` parameter must be set to the **directory** of your experiment's model artifact. The ``name`` argument will be the name used in the Model Registry. You may also pass a dictionary of tags with the ``tags`` parameter. The function returns an object of type :py:class:`mlflow.entities.model_registry.ModelVersion` that contains the registered model's metadata. For example:
+
+.. code-block:: python
+
+    # ... `run_id` and `model_uri` as above
+    model_uri = f"runs:/{run_id}/model"
+    name = "Wine Quality"
+    tags = {"run_id": run_id, "sem_ver": "0.1.0"}
+    registered_model = mlflow.register_model(model_uri=model_uri, name=name, tags=tags)
+    print(f"Name: {registered_model.name}, Version: {registered_model.version}")
+
```

Comment:
Can we also mention that the model can be registered at the same time that a model is logged by specifying the registered model name? (and provide a quick example of that)